src/model/bus.model.ts

```diff
@@ -0,0 +1,3 @@
+export type BusNumber = 'C1' | 'C2' | 'C3' | 'C4' | 'C5' | 'C6' | 'C7' | 'C7ex'
```

Comment:
You can do something like this to avoid copying the values:
```
export const RENNES_LAYERS = [
  'rennesOrtho',
  'rennesBase',
  'metro',
  'bus',
  'bike',
  'trambusLines',
  'trambusStops',
  'parking',
  'poi',
] as const

export type Layers = typeof RENNES_LAYERS[number]
```